frontend/src/api/appwriteApi.js

```diff
@@ -201,6 +195,45 @@ let api = {
 		});
 	},
 
+	/**
+	 * Storage API
+	 **/
+	imageToFileID(imageID) {
+		return imageID && `img${imageID}`;
+	},
+
+	async loadImageFromDatabase(imageID) {
+		const fileID = this.imageToFileID(imageID);
+		try {
+			return await this.sdk.storage.getFile(fileID);
+		} catch(e) {
+			console.error(`Couldn't load image with ID ${fileID} from database! Likely does not exist or was deleted.`);
+			return null;
+		}
+	},
+
+	async saveImageToDatabase(image, imageID) {
+		const fileID = this.imageToFileID(imageID);
+		
+		this.sdk.storage.deleteFile(fileID)
+			.catch(() => console.log(`No image to update. Create new image ${fileID}!`));
+
+		try {
+			const readPermissions = ["*"];
+			console.debug("save file with ID", fileID);
+			await this.sdk.storage.createFile(fileID, image, readPermissions);
```

Comment:
Now I get, why it didn't work. They changed the interface between v0.11 and 0.12 and I used the newest one.

Reply:
Does anyone know how to get the File ID for v0.11?